src/msix/CMakeLists.txt

```diff
@@ -71,19 +72,25 @@ if(WIN32)
     set(Applicability PAL/Applicability/Win32/Applicability.cpp)
 else()
     # Visibility variables for non-win32 platforms
-    set(MSIX_EXPORTS)
+    list(APPEND MSIX_UNPACK_EXPORTS
+        "UnpackPackage"
+        "UnpackPackageFromStream"
+        "UnpackBundle"
+        "UnpackBundleFromStream"
+    )
+    list(APPEND MSIX_UNPACK_EXPORTS
+        "PackPackage"
+    )

```

Comment:
Obviously all platforms will needs similar and for it to be controlled by the pack and/or unpack settings coming in to cmake.

Reply:
Yes, when we add the cmake settings for pack and unpack these lists will only be defined if needed. Also it was MSIX_PACK_EXPORTS not unpack.

---
In reply to: [276763848](https://github.com/Microsoft/msix-packaging/pull/93#discussion_r276763848) [](ancestors = 276763848)